src/storage/kv_storage/littlefs/kv_littlefs.cpp

```diff
@@ -0,0 +1,136 @@
+/**
+ * This software is distributed under the terms of the MIT License.
+ * Copyright (c) 2023 LXRobotics.
+ * Author: Alexander Entinger <alexander.entinger@lxrobotics.com>
+ * Contributors: https://github.com/107-systems/107-Arduino-Cyphal-Support/graphs/contributors.
+ */
+
+/**************************************************************************************
+ * INCLUDE
+ **************************************************************************************/
+
+#include "kv_littlefs.h"
+
+#if __GNUC__ >= 11
```

Comment:
```suggestion
#if !defined(__GNUC__) || (__GNUC__ >= 11)
```

The same applies in the other cases. This is to ensure compatibility with other compilers.

Reply:
I'll add this as an issue and for a next PR. 👍